data-otservbr-global/scripts/quests/ferumbras_ascension/actions_ragiaz.lua

```diff
@@ -0,0 +1,114 @@
+local config = {
```

Comment:
this lever must be done with the new system, as well as the others in `scripts/actions/boss_levers`